tests/affiliatedcertification/tests/affiliated_certification_operator.go

```diff
@@ -1,40 +1,139 @@
+
+			By("Start test")
+
+			err = globalhelper.LaunchTests(
+				[]string{affiliatedcertparameters.AffiliatedCertificationTestSuiteName},
+				affiliatedcertparameters.TestCaseOperatorSkipRegEx,
+			)
+			Expect(err).ToNot(HaveOccurred(), "Error running "+
+				affiliatedcertparameters.AffiliatedCertificationTestSuiteName+" test")
+
+			By("Verify test case status in Junit and Claim reports")
+
+			err = globalhelper.ValidateIfReportsAreValid(
+				affiliatedcertparameters.TestCaseOperatorAffiliatedCertName,
+				globalparameters.TestCaseSkipped)
+			Expect(err).ToNot(HaveOccurred(), "Error validating test reports")
+
+			By("Remove label from operator")
+			err = affiliatedcerthelper.DeleteLabelFromInstalledCSV(
+				affiliatedcertparameters.UncertifiedOperatorPrefixNginx,
+				affiliatedcertparameters.ExistingOperatorNamespace,
+				affiliatedcertparameters.OperatorLabel)
+			Expect(err).ToNot(HaveOccurred(), "Error removing label from operator")
+		})
+
 	// 46582
-	It("one operator to test, operator is certified", func() {
-		err := affiliatedcerthelper.SetUpAndRunOperatorCertTest(
-			[]string{affiliatedcertparameters.CertifiedOperatorApicast}, globalparameters.TestCasePassed)
-		Expect(err).ToNot(HaveOccurred())
+	It("one operator to test, operator belongs to certified-operators organization in Red Hat catalog"+
+		"and its version is certified", func() {
+		By("Deploy operators to test")
+
+		err := affiliatedcerthelper.DeployOperator(affiliatedcertparameters.TestCertificationNameSpace,
+			affiliatedcertparameters.OperatorGroup,
+			affiliatedcertparameters.CertifiedOperatorPostgresSubscription)
+		Expect(err).ToNot(HaveOccurred(), "Error deploying operator")
+
+		By("Confirm that operator is installed and ready")
+
+		Eventually(func() bool {
+			err = affiliatedcerthelper.IsOperatorInstalled(affiliatedcertparameters.TestCertificationNameSpace, "pgo")
+
+			return err == nil
+		}, 5*time.Minute, 5*time.Second).Should(Equal(true), "Operator is not ready")
+
+		By("Label operator to be certified")
+
+		err = affiliatedcerthelper.AddLabelToInstalledCSV(
+			affiliatedcertparameters.CertifiedOperatorPrefixPostgres,
+			affiliatedcertparameters.TestCertificationNameSpace,
+			affiliatedcertparameters.OperatorLabel)
+		Expect(err).ToNot(HaveOccurred(), "Error labeling operator")
+
+		By("Start test")
+
+		err = globalhelper.LaunchTests(
+			[]string{affiliatedcertparameters.AffiliatedCertificationTestSuiteName},
+			affiliatedcertparameters.TestCaseOperatorSkipRegEx,
+		)
+		Expect(err).ToNot(HaveOccurred(), "Error running "+
+			affiliatedcertparameters.AffiliatedCertificationTestSuiteName+" test")
+
+		By("Verify test case status in Junit and Claim reports")
+
+		err = globalhelper.ValidateIfReportsAreValid(
+			affiliatedcertparameters.TestCaseOperatorAffiliatedCertName,
+			globalparameters.TestCasePassed)
+		Expect(err).ToNot(HaveOccurred(), "Error validating test reports")
+
+		By("Remove label from operator")
```

Comment:
This part will be skipped in case if test failed on some line above. Please use AfterEach feature